htdocs/modulebuilder/template/core/modules/modMyModule.class.php

```diff
@@ -179,8 +182,7 @@ public function __construct($db)
 			$conf->mymodule->enabled = 0;
 		}
 
-		// Array to add new pages in new tabs
-		$this->tabs = array();
```

Comment:
Removing this is required ?

Reply:
Like for Permissions, I've moved it after the examples, because i thought seeing some pattern but it seems i've only seen it for Dictionnaries